code/addons/docs/package.json

```diff
@@ -100,6 +100,7 @@
   "dependencies": {
     "@jest/transform": "^29.3.1",
     "@mdx-js/react": "^2.1.5",
+    "@rollup/pluginutils": "^5.0.2",
```

Comment:
Could this be moved to a devDependency (and thus be bundled in)?, therefore not expose this dependency to users.

Reply:
Or is this a peerDependency in some way?